src/app/users/users.service.ts

```diff
@@ -25,4 +25,40 @@ export class UsersService {
     return this.http.get('/users');
   }
 
+  /**
+   * @param {any} user User to be created.
+   * @returns {Observable<any>}
+   */
+  createUser(user: any): Observable<any> {
+    return this.http.post('/users', user);
+  }
+
+  /**
+   * @returns {Observable<any>} Offices data
+   */
+  getOffices(): Observable<any> {
+    return this.http.get('/offices');
+  }
+
+  /**
+   * @param {any} officeId ID of office to retrieve staff from.
+   * @returns {Observable<any>} Staff.
+   */
+  getStaff(officeId: any = ''): Observable<any> {
```

Comment:
It is not staffs here as staffs is not a word (plural for staff is staff), and individual staff do not need to be fetched in this component so I think it is reasonable to not have an 's' at the end.